app/scripts/views/sign_up.js

```diff
@@ -36,6 +40,44 @@ define(function (require, exports, module) {
     return null;
   }
 
+  var actions = {
```

Comment:
I find the `actions` object a bit strange, you could just use three separate functions w/o the need to bundle them.